src/libDirectoryService/ViewChangePostProcessing.cpp

```diff
@@ -216,6 +216,23 @@ void DirectoryService::ProcessViewChangeConsensusWhenDone() {
                   << "newLeaderNetworkInfo: " << newLeaderNetworkInfo);
   }
 
+  // TODO: Refine this
+  // Broadcasting vcblock to lookup nodes
+
+  // Store to blockLink
+  uint64_t latestInd = m_mediator.m_blocklinkchain.GetLatestIndex() + 1;
+  m_mediator.m_blocklinkchain.AddBlockLink(
+      latestInd, m_pendingVCBlock->GetHeader().GetVieWChangeDSEpochNo(),
+      BlockType::VC, m_pendingVCBlock->GetBlockHash());
+
+  vector<unsigned char> dst;
+  m_pendingVCBlock->Serialize(dst, 0);
+
+  if (!BlockStorage::GetBlockStorage().PutVCBlock(
+          m_pendingVCBlock->GetBlockHash(), dst)) {
+    LOG_GENERAL(WARNING, "Unable to put VC Block");
+  }
```

Comment:
Should we return here?

Reply:
I dont think the inability to store should hamper the flow of execution,.

Reply:
Ok